### ISSUE TITLE
The import fails using sugarss syntax

### ISSUE THREAD

**Issue by esiao** [jonathantneal/postcss-partial-import#16]
The syntax sugarss which could be the sass for postcss and still being developed by @ai
Is using a syntax similar to sass and gets rid of semicolons. Which is an issue with this plugin since the parser looks for semicolons and won't import the file if there's one missing.

Opened an issue on sugarss repo which leads me here
https://github.com/postcss/sugarss/issues/13#event-588554352

I think that this plugin is more encline to be the one to be compatible with sugarss since using precss with it makes a lot of sense and it's even stated on the syntax project readme.

https://github.com/postcss/sugarss/

The two other plugins which could handle this are https://github.com/postcss/postcss-import by @MoOx and https://github.com/trysound/postcss-easy-import by @TrySound. In this case I think the later makes more sense if it couldn't be supported by this plugin.

Falling back to .css files for now :)

**Comment by TrySound** [comment]
@esiao First of all stop using cssnext (which is deprecated) in favour of postcss-cssnext.

**Comment by esiao** [comment]
@TrySound it's the case I just don't include it in the plugin name
`var cssnext = require('postcss-cssnext');`

But I don't this it's related, I did my tests by just enabling the import plugins one after the other and commenting all the others plugins just to see if the files were imported.

**Comment by TrySound** [comment]
@esiao `{compress:true}` doesn't exist in `postcss-cssnext`. Which version of postcss-import did you try?

**Comment by esiao** [comment]
@TrySound I made my tests using it like this : 

```
var plugins = [
  precss(), || atImport(), || atEasyImport({prefix: false, extensions: ['.sss'], glob: false})
  // short(),
  // cssnext({compress:true}),
  // lost(),
  // calc(),
  // reporter()
];
```

For the versions : 

```
"postcss-import": "^8.0.2"
"postcss-easy-import": "^1.0.1",
"precss": "^1.4.0"
```

**Comment by MoOx** [comment]
if precss use partial-import first, using others plugins for this is useless, use those first.

**Comment by esiao** [comment]
To clarify for everyone, this is exactly what I did.
I first tried using only precss. It didn't work as expected with the missing semicolon error.
I then tried using postcss-import, didn't work too because of the missing semicolon warning.
And lastly tried postcss-easy-import which did fail too because of the missing semicolon warning.

The issue here is only that the importers expects to have valid CSS for the Parser when importing which is not the case because .sss is using a sass like syntax without semicolons.

I first thought it was an issue with sugarss execution timeline, the parser should do it's job before the plugins to provide valid css. But this doesn't seem to be the direction @ai is wanting, it's more to the plugin to adapt their parsers based on the parser option or the extension name.
